src/migrations/20221121133546-soft-delete-user.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+exports.up = function (db, callback) {
+    db.runSql(
+        `
+            ALTER TABLE users ADD COLUMN IF NOT EXISTS is_deleted boolean not null default false;
```

Comment:
it's better to have deleted_at with date because it opens more options in future